fiftyone/brain/internal/core/leaky_splits.py

```diff
@@ -0,0 +1,497 @@
+"""
+Finds leaks between splits.
+"""
+
+from collections import defaultdict
+from copy import copy
+
+import fiftyone as fo
+from fiftyone import ViewField as F
+
+# pylint: disable=no-member
+import cv2
+
+import eta.core.utils as etau
+
+import fiftyone.core.brain as fob
+import fiftyone.brain.similarity as sim
+import fiftyone.brain.internal.core.sklearn as skl_sim
+import fiftyone.brain.internal.core.duplicates as dups
+import fiftyone.brain.internal.core.utils as fbu
+import fiftyone.core.utils as fou
+import fiftyone.core.validation as fov
+import fiftyone.zoo as foz
+
+fbu = fou.lazy_import("fiftyone.brain.internal.core.utils")
+
+_DEFAULT_MODEL = "clip-vit-base32-torch"
+_DEFAULT_BATCH_SIZE = None
+
+
+def compute_leaky_splits(
```

Comment:
I agree with @mwoodson1 that, in order to ship this as part of the Brain, it should be exposed at the top-level via `fob.compute_leaky_splits()` or similar.

Reply:
I agree, it needs to follow the pattern `fob.compute_X()` that other Brain methods do. Preference for calling it `compute_leaks()`, rather than `compute_leaky_splits()`

Reply:
This is just a matter of exposing this function in the `__init__`, correct?